public/javascripts/data-ui.js

```diff
@@ -24,9 +24,19 @@ var dataNS = odkmaker.namespace.load('odkmaker.data');
             },
             error: function(request, status, error)
             {
+                console.log(error);
+                
+                if (error == "Not Found") 
+                {
+                    var msg = '<p>Error: No form selected.</p>'
+                }
+                else{
+                    var msg = '<p>Could not open the form. See browser console log for a detailed error message.</p>';
+                    console.log(error);
```

Comment:
This is redundant to the `console.log` outside of the condition, right?

Reply:
Ah, I missed that one, yes I'll drop L35.

Reply:
Redundant console.log removed in https://github.com/getodk/build/pull/285/commits/739289fdf6a7766cdd370b570d95a9fdec232aab.